### PR TITLE
fix: Speaker Image in Session detailed segment is good now

### LINKED ISSUE #3756: Public Event Page: Speaker Image in Session detailed segment is blurred and pixelated
**Describe the bug**

Speaker Image in Session detailed segment is blurred and pixelated.

**To Reproduce**

Steps to reproduce the behaviour:

1. Confirm a Speaker and Session Proposal
2. Go on public event page and click on session
3. Click on the session and see the speaker image

**Expected behaviour**

Image should not be pixelated.

**Screenshots**

919953116575_status_f9713d2cc7c94122bfb28f6aec947b17

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes #3756 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests, and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

Screenshot from 2020-01-04 20-47-59

### CODE REVIEW

**Comment by maze-runnar** [comment]
@kushthedude @uds5501 please review .

File: app/templates/components/public/session-item.hbs [hunk 1]
```diff
@@ -44,7 +44,7 @@
   <div class="content">
     {{#each session.speakers as |speaker|}}
       <div class="ui divider"></div>
-      <img alt="speaker" class="ui tiny avatar image" src="{{if speaker.iconImageUrl speaker.iconImageUrl (if speaker.photoUrl speaker.photoUrl '/images/placeholders/avatar.png')}}">
+      <img alt="speaker" class="ui avatar image" src="{{if speaker.iconImageUrl speaker.iconImageUrl (if speaker.photoUrl speaker.photoUrl '/images/placeholders/avatar.png')}}">
```
Comment (kushthedude) [inline app/templates/components/public/session-item.hbs]: You have just changed the size of image. For the given example even tiny class will give the appropriate image.

Comment (maze-runnar) [reply app/templates/components/public/session-item.hbs]: oops ! i just missed that , sorry . So what is the difference between image i uploaded and the image that is showing blurred .

Comment (maze-runnar) [reply app/templates/components/public/session-item.hbs]: As it's showing fine for 3-4 images i uploaded .

Comment (maze-runnar) [reply app/templates/components/public/session-item.hbs]: `speaker`

why this line is there in code , there is no speaker icon  :   **if speaker.iconImageUrl speaker.iconImageUrl**

Comment (iamareebjamal) [reply app/templates/components/public/session-item.hbs]: There is

**Comment by maze-runnar** [comment]
Screenshot from 2020-01-04 22-54-43
for one more image it's fine .

**Comment by maze-runnar** [comment]
@kushthedude can you please check it once on your local system .  I have tried several images but the quality remains the same .

**Comment by iamareebjamal** [comment]
Because you are not using celery

File: app/templates/components/public/session-item.hbs [hunk 2]
```diff
@@ -44,7 +44,7 @@
   <div class="content">
     {{#each session.speakers as |speaker|}}
       <div class="ui divider"></div>
-      <img alt="speaker" class="ui tiny avatar image" src="{{if speaker.iconImageUrl speaker.iconImageUrl (if speaker.photoUrl speaker.photoUrl '/images/placeholders/avatar.png')}}">
+      <img alt="speaker" class="ui avatar small image" src="{{if speaker.iconImageUrl speaker.iconImageUrl (if speaker.photoUrl speaker.photoUrl '/images/placeholders/avatar.png')}}">
```
Comment (iamareebjamal) [inline app/templates/components/public/session-item.hbs]: There is no need to change class, change iconImage to larger image variant

**Comment by maze-runnar** [comment]
@kushthedude @uds5501 i downloaded the same image as described in issue  and put it there but 
Screenshot from 2020-01-05 00-07-04

**Comment by maze-runnar** [comment]
> celery

So how to get it ?

File: app/templates/components/public/session-item.hbs (hunk 2 above)
Comment (maze-runnar) [reply app/templates/components/public/session-item.hbs]: ok

File: app/templates/components/public/session-item.hbs (hunk 1 above)
Comment (maze-runnar) [reply app/templates/components/public/session-item.hbs]: Done :+1:

File: app/templates/components/public/session-item.hbs (hunk 2 above)
Comment (iamareebjamal) [reply app/templates/components/public/session-item.hbs]: Use thumbnailImageUrl instead

**Comment by kushthedude** [comment]
> > celery
> 
> So how to get it ?

You can see the guidelines for following in the open event server repository

**Comment by kushthedude** [comment]
Superseded by #3765 .
@maze-runnar The release was on hold due to the following hence the new PR was made.
Thanks

**Pull request closed without merging**